Feature/add event 2018

cristofer-dev opened the pull request:
- Borrador del Sitio conferencia 2018
- @CepheiA agregar los planes

DMaster69 commented:
@hpaez en la sección "prensa", mantén el orden de las fechas, la noticia más nueva al inicio.

hpaez commented:
ya realice el cambio, creo que se encuentra en el pull request, si no indicamelo

cristofer-dev committed:
UPDATE Banner Nija Developers

cristofer-dev committed:
ADD Section DevConferenceValley

	modified:   _includes/head.html
	new file:   _layouts/devConf.html
	new file:   assets/css/bg_stars.css
	modified:   assets/css/main.css
	new file:   assets/img/LogoDevConf2017.png
	new file:   assets/img/logo_h_blanco.png
	new file:   assets/img/stars.png
	new file:   assets/img/twinkling.png
	new file:   evento2017.html

cristofer-dev committed:
Update Menor

cristofer-dev committed:
Update CSS y Logos + Section Partners

	modified:   _layouts/devConf.html
	modified:   assets/css/main.css
	new file:   assets/img/partners/logo_labrujula_white.png
	new file:   assets/img/partners/logo_ust_white.png
	modified:   evento2017.html

DMasterIO committed:
Update evento2017.html

cristofer-dev committed:
ADD Content Section DevConference2017

	modified:   assets/css/main.css
	modified:   evento2017.html

cristofer-dev committed:
Merge branch 'master' of https://github.com/iv-devs/iv-devs.github.io

cristofer-dev committed:
MODIFY Textos Section DevConferenceValley2017

	modified:   assets/css/main.css
	modified:   evento2017.html

cristofer-dev committed:
FIX paddin rows and format p

	modified:   assets/css/main.css
	modified:   evento2017.html

cristofer-dev committed:
MODIFY font-size em to vh section DevConferenceValley

	modified:   assets/css/main.css

cristofer-dev committed:
UPDATE texts sectioin DevConferenceValley

	modified:   evento2017.html

cristofer-dev committed:
ADD Logo Noders White

	renamed:    assets/img/partners/logo_noder.png -> assets/img/partners/logo_noders_black.png
	new file:   assets/img/partners/logo_noders_white.png
	modified:   evento2017.html

cristofer-dev committed:
MODIFY minor to css

	modified:   assets/css/main.css
	modified:   evento2017.html

cristofer-dev committed:
UPDATE HOME Banner DevConferenceValley

	new file:   assets/img/meetups/DevConference2017.png
	modified:   index.html

cristofer-dev committed:
FIX Home Logo Noders

	modified:   index.html

cristofer-dev committed:
UPDATE Readme.MD

 Changes to be committed:
	modified:   README.md

diruzcode committed:
Change Name Programadores Chile > ProinChile

Change name & Change Logo

DMasterIO committed:
Merge pull request #24 from DiruzCode/patch-1

Change Name Programadores Chile > ProinChile

diruzcode committed:
change name and img logo

change name and img logo

DMasterIO committed:
Merge pull request #25 from DiruzCode/patch-2

change name and img logo

Commit:
UPDATE Logo UST DevConferenceValley

 Changes to be committed:
	new file:   assets/img/partners/logo_ust_informatica_white.png
	modified:   evento2017.html

Commit:
UPDATE LOGO AI UST

 Changes to be committed:
	modified:   assets/img/partners/logo_ust_informatica_white.png

Commit:
UPDATE Banner HOME para DevConferenceValley

 Changes to be committed:
	modified:   assets/img/meetups/DevConference2017.png

Commit:
UPDATE Section DevConferencevalley : Charlas

 Changes to be committed:
	modified:   _layouts/devConf.html
	new file:   assets/img/DevConferenceValley2017/1.png
	new file:   assets/img/DevConferenceValley2017/2.png
	new file:   assets/img/DevConferenceValley2017/3.png
	new file:   assets/img/DevConferenceValley2017/4.png
	new file:   assets/img/DevConferenceValley2017/5.png
	new file:   assets/img/DevConferenceValley2017/6.png
	new file:   assets/img/DevConferenceValley2017/7.png
	modified:   evento2017.html

Commit:
UPDATE img DevConferenceValley

 Changes to be committed:
	modified:   assets/img/DevConferenceValley2017/1.png
	modified:   assets/img/DevConferenceValley2017/2.png
	modified:   assets/img/DevConferenceValley2017/3.png
	modified:   assets/img/DevConferenceValley2017/4.png
	modified:   assets/img/DevConferenceValley2017/5.png
	modified:   assets/img/DevConferenceValley2017/6.png
	modified:   assets/img/DevConferenceValley2017/7.png
	modified:   evento2017.html

Commit:
UPDATE HOME Habilitacion Tickets Evento 2017

 Changes to be committed:
	modified:   index.html

Commit:
UPDATE Banners Charlistas

 Changes to be committed:
	modified:   assets/img/DevConferenceValley2017/1.png
	modified:   assets/img/DevConferenceValley2017/2.png
	modified:   assets/img/DevConferenceValley2017/3.png
	modified:   assets/img/DevConferenceValley2017/4.png
	modified:   assets/img/DevConferenceValley2017/5.png
	modified:   assets/img/DevConferenceValley2017/7.png

cristofer-dev committed:
UPDATE Home

 Changes to be committed:
	modified:   index.html

cristofer-dev committed:
UPDATE Banner 6

 Changes to be committed:
	modified:   assets/img/DevConferenceValley2017/6.png

cristofer-dev committed:
UPDATE Banne IA : cancelada

 Changes to be committed:
	modified:   assets/img/DevConferenceValley2017/2.png

cristofer-dev committed:
UPDATE Section Sponsors

 Changes to be committed:
	modified:   assets/css/main.css
	new file:   assets/img/DevConferenceValley2017/sponsor1.png
	new file:   assets/img/DevConferenceValley2017/sponsor2.png
	new file:   assets/img/DevConferenceValley2017/sponsor3.png
	new file:   assets/img/DevConferenceValley2017/sponsor4.png
	new file:   assets/img/DevConferenceValley2017/sponsor5.png
	new file:   assets/img/DevConferenceValley2017/sponsor6.png
	new file:   assets/img/partners/logo_microsoft.png
	modified:   evento2017.html

cristofer-dev committed:
ADD Meetup 2017-09 Scrum

 Changes to be committed:
	new file:   assets/img/meetups/2017-09-02-scrum.jpg
	modified:   index.html

hpaez committed:
UPDATE section Events

cristofer-dev committed:
Merge pull request #26 from hpaez/feature/update-section-events

UPDATE section Events

hpaez committed:
ADD license MIT

cristofer-dev committed:
Merge pull request #27 from hpaez/feature/add-license-mit

ADD license MIT

hpaez committed:
ADD NEW SECTION IN EVENT AND ADD IMG EVENT AND MEETUP

cristofer-dev committed:
Merge pull request #28 from hpaez/feature/add-new-other-section-in-events

ADD NEW SECTION IN EVENT AND ADD IMG EVENT AND MEETUP

hpaez committed:
Revert "ADD NEW SECTION IN EVENT AND ADD IMG EVENT AND MEETUP"

hpaez committed:
ADD NEW SECTION IN EVENT AND ADD IMG EVENT AND MEETUP

cristofer-dev committed:
Merge pull request #29 from hpaez/revert-28-feature/add-new-other-section-in-events

Revert "ADD NEW SECTION IN EVENT AND ADD IMG EVENT AND MEETUP"

cristofer-dev committed:
Merge pull request #30 from hpaez/feature/add-new-other-section-in-events

ADD NEW SECTION IN EVENT AND ADD IMG EVENT AND MEETUP

cristofer-dev committed:
UPDATE Home && Evento2017

 Changes to be committed:
	modified:   evento2017.html
	modified:   index.html

cristofer-dev committed:
UPDATE HOME ADD Meetup Octubre 2017

 Changes to be committed:
	new file:   assets/img/meetups/2017-10-07-ReactNative.jpg
	modified:   index.html

cristofer-dev committed:
UPDATE Eventos

 Changes to be committed:
	new file:   assets/img/eventos/2017-04-11-hangouts-Fernando-larranaga.jpg
	renamed:    assets/img/meetups/2017-10-07-ReactNative.jpg -> assets/img/eventos/2017-10-07-ReactNative.jpg
	modified:   eventos.html
	modified:   index.html

cristofer-dev committed:
REMOVE Banner Meetup Octubre

 Changes to be committed:
	modified:   index.html

cristofer-dev committed:
UPDATE Curso python desde cero

 Changes to be committed:
	new file:   assets/img/eventos/2017-11-25-Python_clase1.png
	modified:   index.html

TODO: Linkear a meetup.com

cristofer-dev committed:
UPDATE Banner python

 Changes to be committed:
	modified:   assets/img/eventos/2017-11-25-Python_clase1.png

cristofer-dev committed:
Enable Button "Inscribirme"

 Changes to be committed:
	modified:   index.html

cristofer-dev committed:
UPDATE Banner Python

 Changes to be committed:
	new file:   assets/img/eventos/2017-11-25-Python_claseN.png
	modified:   index.html

hpaez committed:
Arregla problema al visualizar descripciones largas en los post del blog

Lo que soluciona este commit, es el hecho de agrupar de a 2 post para que en el caso de hablar post con descripcion larga salga visible de mejor manera.

DMasterIO committed:
Merge pull request #31 from hpaez/feature/arregla-visualizacion-post-blog

Arregla problema al visualizar descripciones largas en los post del blog

cristofer-dev committed:
Publish site conf 2017

cristofer-dev committed:
ADD draft landing conference 2018

 Changes to be committed:
	modified:   evento2017.html
	new file:   evento2018.html
